mkdocs.yml

```diff
@@ -0,0 +1,6 @@
+site_name: Entwicklerdokumentation
```

Comment:
Site name should be in English like the whole documentation and additional empty lines should be removed.